packages/runtime/runtime-utils/src/summaryUtils.ts

```diff
@@ -228,15 +234,29 @@ export class SummaryTreeBuilder implements ISummaryTreeWithStats {
 		this.summaryStats.handleNodeCount++;
 	}
 
+	/**
+	 *
```

Comment:
All our doc comments should have at least a minimal top level description (thats not part of any tags) since the way our API docs are displayed on the site relies on that description to provide a summary. Note that when its missing, API-Documenter reports the type as "undocumented" which helps detect that.

```suggestion
	 * Adds a child and updates the stats accordingly.
```